yt/frontends/parthenon/fields.py

```diff
@@ -0,0 +1,165 @@
+        # Magnetic fields carry units of 1/sqrt(pi) so we cannot directly forward
+        # and need to setup aliases below.
+        ("prim_magnetic_field_1", (mag_units, [], None)),
+        ("prim_magnetic_field_2", (mag_units, [], None)),
+        ("prim_magnetic_field_3", (mag_units, [], None)),
+        # New field naming (i.e., "variable_component") of conserved variables
+        ("cons_density", (rho_units, ["density"], None)),
+        ("cons_momentum_density_1", (mom_units, ["momentum_density_x"], None)),
+        ("cons_momentum_density_2", (mom_units, ["momentum_density_y"], None)),
+        ("cons_momentum_density_3", (mom_units, ["momentum_density_z"], None)),
+        ("cons_total_energy_density", (eng_units, ["total_energy_density"], None)),
+        # Magnetic fields carry units of 1/sqrt(pi) so we cannot directly forward
+        # and need to setup aliases below.
+        ("cons_magnetic_field_1", (mag_units, [], None)),
+        ("cons_magnetic_field_2", (mag_units, [], None)),
+        ("cons_magnetic_field_3", (mag_units, [], None)),
+        # Legacy naming. Given that there's no conflict with the names above,
+        # we can just define those here so that the frontend works with older data.
+        ("Density", (rho_units, ["density"], None)),
+        ("Velocity1", (mom_units, ["velocity_x"], None)),
+        ("Velocity2", (mom_units, ["velocity_y"], None)),
+        ("Velocity3", (mom_units, ["velocity_z"], None)),
+        ("Pressure", (pres_units, ["pressure"], None)),
+        ("MagneticField1", (mag_units, [], None)),
+        ("MagneticField2", (mag_units, [], None)),
+        ("MagneticField3", (mag_units, [], None)),
+        ("MomentumDensity1", (mom_units, ["momentum_density_x"], None)),
+        ("MomentumDensity2", (mom_units, ["momentum_density_y"], None)),
+        ("MomentumDensity3", (mom_units, ["momentum_density_z"], None)),
+        ("TotalEnergyDensity", (eng_units, ["total_energy_density"], None)),
+    )
+
+    def setup_fluid_fields(self):
+        from yt.fields.magnetic_field import setup_magnetic_field_aliases
+
+        unit_system = self.ds.unit_system
+        # Add velocity fields (if only momemtum densities are given)
+        for i, comp in enumerate(self.ds.coordinates.axis_order):
+            # Support both current and legacy scheme
+            for mom_field_name in ["MomentumDensity", "cons_momentum_density_"]:
+                mom_field = ("parthenon", f"{mom_field_name}{i+1}")
+                if mom_field in self.field_list:
+                    self.add_field(
+                        ("gas", f"velocity_{comp}"),
+                        sampling_type="cell",
+                        function=velocity_field(mom_field),
+                        units=unit_system["velocity"],
+                    )
+        # Figure out thermal energy field
+        if ("parthenon", "Pressure") in self.field_list or (
+            "parthenon",
+            "prim_pressure",
```

Comment:
I think formatting would end up less terrible here if we removed this magic comma
```suggestion
            "prim_pressure"
```

Reply:
My formatter makes this magic comma come back again and again.

Reply:
Ok I won't go into full bikeshedding mode here (the point of using formatters is to save us from having to discussing it), but just in case it's relevant, I'd like to point out that you should be using the formatting toolbelt that is configured by `pre-commit-config.yml`, otherwise conflicts are inevitable. Are you ?

Reply:
alright, feel free to reslove these threads then